db/migrate/20170327100309_lock_plugin_roles.rb

```diff
@@ -0,0 +1,15 @@
+class LockPluginRoles < ActiveRecord::Migration
+  def up
+    Foreman::Plugin.all.each do |plugin|
```

Comment:
I think we can't do this by migration, if I add plugin later after this migration was already done, it would have roles unlocked... I think we need to do it at runtime when we're adding permissions to the role